pkg/controllers/v1alpha1/databackup/implement.go

```diff
@@ -201,7 +201,37 @@ func (r *DataBackupReconcilerImplement) reconcilePendingDataBackup(ctx reconcile
 func (r *DataBackupReconcilerImplement) reconcileExecutingDataBackup(ctx reconcileRequestContext) (ctrl.Result, error) {
 	log := ctx.Log.WithName("reconcileExecutingDataBackup")
 	// 1. get the alluxio-master Pod
-	podName := ctx.Dataset.Name + "-master-0"
+	// For HA Mode, need find the leading master pod name
+	targetDataset := ctx.Dataset
+	index, boundedRuntime := utils.GetRuntimeByCategory(targetDataset.Status.Runtimes, common.AccelerateCategory)
+	if index == -1 {
+		log.Info("bounded runtime with Accelerate Category is not found on the target dataset", "targetDataset", targetDataset)
+	}
+
+	var podName string
+	if boundedRuntime.MasterReplicas <= 1 {
+		podName = ctx.Dataset.Name + "-master-0"
```

Comment:
It is better to define it uniformly through common pkg

Reply:
SGTM